doc/quick-install.md

```diff
@@ -1,162 +1,3 @@
-
-If you used a self-signed certificate (as opposed to using Let's Encrypt), your web browser will ask you to confirm that you trust the certificate.
-
-You should see a web page prompting you to create a new meeting.  Make sure that you can successfully create a meeting and that other participants are able to join the session.
-
-If this all worked, then congratulations!  You have an operational Jitsi conference service.
-
-## Adding sip-gateway to Jitsi Meet
-
-### Install Jigasi
-
-Jigasi is a server-side application acting as a gateway to Jitsi Meet conferences. It allows regular [SIP](https://en.wikipedia.org/wiki/Session_Initiation_Protocol) clients to join meetings and provides transcription capabilities.
-
-```sh
-sudo apt install jigasi
-```
-
-During the installation, you will be asked to enter your SIP account and password. This account will be used to invite the other SIP participants.
-
-### Reload Jitsi Meet
-
-Launch again a browser with the Jitsi Meet URL and you'll see a telephone icon on the right end of the toolbar. Use it to invite SIP accounts to join the current conference.
-
-Enjoy!
-
-## Uninstall
-
-```sh
-sudo apt purge jigasi jitsi-meet jitsi-meet-web-config jitsi-meet-prosody jitsi-meet-turnserver jitsi-meet-web jicofo jitsi-videobridge2
-```
-
-Sometimes the following packages will fail to uninstall properly:
-
-- jigasi
-- jitsi-videobridge
-
-When this happens, just run the uninstall command a second time and it should be ok.
-
-The reason for the failure is that sometimes the uninstall script is faster than the process that stops the daemons. The second run of the uninstall command fixes this, as by then the jigasi or jitsi-videobridge daemons are already stopped.
-
-#### Systemd details
-To reload the systemd changes on a running system execute `sudo systemctl daemon-reload` and `sudo systemctl restart jitsi-videobridge2`.
-To check the tasks part execute `sudo systemctl status jitsi-videobridge2` and you should see `Tasks: XX (limit: 65000)`.
-To check the files and process part execute ```cat /proc/`cat /var/run/jitsi-videobridge/jitsi-videobridge.pid`/limits``` and you should see:
-```
-Max processes             65000                65000                processes
-Max open files            65000                65000                files
-```
-
-## Debugging problems
-
-If you run into problems, one thing to try is using a different web browser. Some versions of some browsers are known to have issues with Jitsi Meet. You can also visit https://test.webrtc.org to test your browser's [WebRTC](https://en.wikipedia.org/wiki/WebRTC) support.
-
-Another place to look is the various log files:
-
-```
-/var/log/jitsi/jvb.log
-/var/log/jitsi/jicofo.log
-/var/log/prosody/prosody.log
-```
+This document has been moved [here](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart).
```

Comment:
Have these proposals been cherry-picked and reflected to this new handbook?
https://github.com/jitsi/jitsi-meet/pulls?q=is%3Apr+is%3Aopen+quick-install.md

Reply:
Yes the plan is to move all PRs there. If you want to help with that, awesome!

Reply:
Right, please let me take care of that. I'm cherry-picking them and fixing conflicts as many as possible 😄